config.md

```diff
@@ -229,17 +229,23 @@ _Note: For Solaris, uid and gid specify the uid and gid of the process inside th
 ## Hooks
 
 Lifecycle hooks allow custom events for different points in a container's runtime.
-Presently there are `Prestart`, `Poststart` and `Poststop`.
 
-* [`Prestart`](#prestart) is a list of hooks to be run before the container process is executed
-* [`Poststart`](#poststart) is a list of hooks to be run immediately after the container process is started
-* [`Poststop`](#poststop) is a list of hooks to be run after the container process exits
+* **`hooks`** (object, optional) MAY contain any of the following properties:
+  * **`prestart`** (array, optional) is an array of [pre-start hooks](#prestart).
+    Entries in the array contain the following properties:
+    * **`path`** (string, required) with the same semantics as Go's [`Cmd.Path`][go-cmd].
+      Hook paths are absolute and are executed from the host's filesystem in the [runtime namespace][runtime-namespace].
+    * **`args`** (array of strings, optional) with the same semantics as Go's [`Cmd.Args`][go-cmd].
```

Comment:
On Fri, May 06, 2016 at 10:36:29AM -0700, Michael Crosby wrote:
> > +    * **`path`** (string, required) with the same semantics as Go's [`Cmd.Path`][go-cmd].
> > +      Hook paths are absolute and are executed from the host's filesystem in the [runtime namespace][runtime-namespace].
> > +    * **`args`** (array of strings, optional) with the same semantics as Go's [`Cmd.Args`][go-cmd].
> 
> Don't make this go specific, its args and env like every args and env ever

The Go reference is inherited from 48049d2 (Clarify the semantics of
hook elements, 2015-11-25, #255).  I'm fine changing it to an
execve(2) reference or some such if you'd rather, but folks might
consider that platform-specific.  POSIX also has execve(3p) [1], but
it's in unistd.h, which may be POSIX-specific.  Is there a generic C
spec that covers this idea?

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/functions/fexecve.html